service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocmosisCoverDocumentsServiceTest.java

```diff
@@ -1,31 +1,36 @@
 package uk.gov.hmcts.reform.fpl.service;
 
-import com.google.common.collect.ImmutableMap;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 import uk.gov.hmcts.reform.fpl.model.Address;
 import uk.gov.hmcts.reform.fpl.model.Representative;
 import uk.gov.hmcts.reform.fpl.model.common.DocmosisDocument;
-
-import java.util.Map;
+import uk.gov.hmcts.reform.fpl.model.docmosis.DocmosisCoverDocument;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.BDDMockito.given;
 
 @ExtendWith(SpringExtension.class)
-@ContextConfiguration(classes = {DocmosisCoverDocumentsService.class})
+@ContextConfiguration(classes = { JacksonAutoConfiguration.class, DocmosisCoverDocumentsService.class })
 class DocmosisCoverDocumentsServiceTest {
+    private static final String NULL_FAMILY_MAN_NUMBER = null;
+    private static final String FAMILY_MAN_NUMBER = "12345";
+    private static final Long CCD_CASE_NUMBER = 1234123412341234L;
 
-    private final DocmosisCoverDocumentsService documentsService;
+    private byte[] pdf = { 1, 2, 3, 4, 5 };
+    private DocmosisDocument docmosisDocument = new DocmosisDocument("example.pdf", pdf);
+    private Representative testRepresentative = buildRepresentative();
+    private DocmosisCoverDocumentsService documentsService;
 
     @MockBean
-    DocmosisDocumentGeneratorService documentGeneratorService;
+    private DocmosisDocumentGeneratorService documentGeneratorService;
 
     @Autowired
     DocmosisCoverDocumentsServiceTest(DocmosisCoverDocumentsService documentsService) {
```

Comment:
in this can we not just autowire DocmosisCoverDocumentsService rather than having a constructor in the test?